var/spack/repos/builtin/packages/swig/package.py

```diff
@@ -20,22 +20,31 @@ class Swig(AutotoolsPackage):
        application."""
 
     homepage = "http://www.swig.org"
-    url      = "http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz"
+    git = "https://github.com/swig/swig"
 
-    version('3.0.12', '82133dfa7bba75ff9ad98a7046be687c')
-    version('3.0.11', '13732eb0f1ab2123d180db8425c1edea')
-    version('3.0.10', 'bb4ab8047159469add7d00910e203124')
-    version('3.0.8',  'c96a1d5ecb13d38604d7e92148c73c97')
-    version('3.0.2',  '62f9b0d010cef36a13a010dc530d0d41')
-    version('2.0.12', 'c3fb0b2d710cc82ed0154b91e43085a4')
-    version('2.0.2',  'eaf619a4169886923e5f828349504a29')
-    version('1.3.40', '2df766c9e03e02811b1ab4bba1c7b9cc')
+    version('4.0.0', commit='9c3faeb0ee6c588bc4772bd186554796cc1e446e',
+            preferred=True)
+    version('fortran', branch='master',
+            git='https://github.com/swig-fortran/swig.git')
+    version('3.0.12', commit='fbeb566014a1d320df972aef965daf042db7db36')
+    version('2.0.12', commit='18c9eca1827e009c66e5651145aff17745b76140')
+    version('1.3.40', commit='fc52ca3b80434eee001745e93c07cbf0fb7095cf')
 
     depends_on('pcre')
 
+    # Repository does *not* include configure, so it calls autoreconf
+    depends_on('m4')
+    depends_on('autoconf')
+    depends_on('automake')
+    depends_on('libtool')
```

Comment:
Why change to the git repo? Did the release tarballs not build? You can have a `fortran` version that downloads from git and other versions that download via URL.